manifests/gnome/gconftool_2.pp

```diff
@@ -57,7 +65,7 @@
   exec { "gconftool-2 ${key}" :
     command => "gconftool-2 --direct --config-source xml:readwrite:${config_real} --set '${key}' --type ${type_real} '${value_string}'",
     # "2>&1" is needed to catch cases where we want to write an empty string when no value is set (yet)
-    unless  => "test \"$(gconftool-2 --direct --config-source xml:readwrite:${config_real} --get ${key} 2>&1 )\" == \"${value_string}\"",
-    path    => '/bin:/sbin:/usr/bin:/usr/sbin',
+    unless  => "test \"$(gconftool-2 --direct --config-source xml:readwrite:${config_real} --get ${key} | tail -n1 2>&1 )\" == \"${value_string}\"",
```

Comment:
gconftool-2 on SLE10 produces and extra line with:

"Resolved address "xml:readwrite:/etc/opt/gnome/gconf/gconf.xml.mandatory" to a writable configuration source at position 0"